stix_shifter_utils/stix_transmission/utils/RestApiClientAsync.py

```diff
@@ -57,6 +57,10 @@ def __init__(self, host, port=None, headers={}, url_modifier_function=None, cert
         unique_file_handle = uuid.uuid4()
         self.server_cert_name = "/tmp/{0}-server_cert.pem".format(unique_file_handle)
         server_ip = host
+        self.trust_env_enabled = os.environ.get("STIX_SHIFTER_ENABLE_TRUST_ENV", False)
```

Comment:
Should we maybe call this something with PROXY?

Reply:
I wanted it to match the actual setting we are configuring. It does only impact proxy and it is only being used for that, but I prefer it to be clear of what it is changing.